README.md

```diff
@@ -64,6 +64,14 @@ By default, this plugin only compiles ["secure" schemas][secure]. This can be di
 
 By default, only files with a `.schema` extension are compiled. If you have different requirements you can pass a Regexp to `filter` for the plugin to use.
 
+### `formats`
+
+Includes [ajv-formats](https://github.com/ajv-validator/ajv-formats). Default `true`.
+
+### `options`
```

Comment:
This kind of overload the term `options` in the context of the plugin (as opposed to Ajv). Would it be clearer if this was called `ajvOptions`?